### ISSUE TITLE
How to open Mullvad Browser ? 

### ISSUE THREAD

**Issue by Yukinooooooo** [mullvad/mullvad-browser#217]
I'm using Fedora 39 KDE, Kernel Version : 6.6.11, Wayland

I right click to open the terminal option in "Browser" folder from "mullvad-browser" folder, I type `./mullvadbrowser.real`, I got an error message : 

```
[username@fedora Browser] $ ./mullvadbrowser.real
XPCOMGlueLoad error for file /home/username/Documents/mullvad-browser/Browser/libxul.so:
libdbus-glib-1.so.2: cannot open shared object file: No such file or directory
Couldn't load XPCOM.
```

**Comment by Tre-brock** [comment]
I can't help you from launching it from the terminal, but in KDE if you go into the mullvad-browser folder there should be an icon that says mullvad-browser and a folder. Open the app. And if that doesn't work you will have to give it permission to be opened as an executable.

**Comment by Thorin-Oakenpants** [comment]
like this -> https://forum.torproject.org/t/how-to-start-tor-browser-on-linux/11202/8 - blame gnome + nautilus

**Comment by MichaelDevon** [comment]
I'm on Arch, but i fixed this exact thing by installing the "dbus-glib" package.

**Comment by ii7ussieN** [comment]
> I'm on Arch, but i fixed this exact thing by installing the "dbus-glib" package.

Thanks, fixed it for me

**Comment by ruihildt** [comment]
This is a duplicate of #117 and will be fixed in Mullvad Browser 14.0 (based on the next Firefox ESR).
